src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -1060,13 +1122,6 @@ private void resolveToken(ref CORINFO_RESOLVED_TOKEN pResolvedToken)
                 MethodDesc method = result as MethodDesc;
                 pResolvedToken.hMethod = ObjectToHandle(method);
                 pResolvedToken.hClass = ObjectToHandle(method.OwningType);
-
-#if READYTORUN
-            if (tokenContextToRecord != null)
-            {
-                _compilation.NodeFactory.Resolver.AddModuleTokenForMethod(method, new ModuleToken(tokenContextToRecord, (mdToken)pResolvedToken.token));
```

Comment:
Why was this added in the first place? Is removing it going to cause problems?

Reply:
This is moved up. I just figured that since we need to do the MethodDesc/FieldDesc/TypeDesc disambiguation in the code above, it's just better to reduce the number of READYTORUN blocks as well.